presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1351,4 +1352,17 @@ public HiveClientConfig setPushdownFilterEnabled(boolean pushdownFilterEnabled)
         this.pushdownFilterEnabled = pushdownFilterEnabled;
         return this;
     }
+
+    public boolean isSubfieldFilterEnabled()
```

Comment:
Shouldn't this be called subfield pruning or extracting or something?

Reply:
this session property is for enabling filters on subfields (nested columns)